.pipelines/release.yml

```diff
@@ -156,7 +156,7 @@ extends:
           inputs:
             solution: '**\PowerToys.sln'
             vsVersion: 17.0
-            msbuildArgs: -restore /p:RestorePackagesConfig=true /p:RestoreConfigFile="$(Build.SourcesDirectory)\.pipelines\release-nuget.config" /p:CIBuild=true /bl:$(Build.SourcesDirectory)\msbuild.binlog
+            msbuildArgs: -restore /p:RestorePackagesConfig=true /p:RestoreConfigFile="$(Build.SourcesDirectory)\.pipelines\release-nuget.config" /bl:$(Build.SourcesDirectory)\msbuild.binlog
```

Comment:
Why are we removing this flag? That way installers will try to Install a certificate for VCM, which is not needed for official installs since the driver is signed with a Microsoft certificate.

Reply:
Actually the important ones are on installer-steps.yml

Reply:
Yea I couldn't find anything in the repo which referenced "CIBuild" so removed it. Well, except an `#ifdef`, but MSBuild properties don't translate to those anyway, so that wouldn't actually do anything.

Unless there's some magic related to this property somewhere I'm just not seeing.

Reply:
I'll add it back anyway though since at worst it's just benign.